conf/ocsci/submariner_downstream.yaml

```diff
@@ -0,0 +1,4 @@
+ENV_DATA:
+  submariner_source: "downstream"
+DEPLOYMENT:
+  ui_deployment: true
```

Comment:
As discussed below, remove.

It might cause and affect OCS/ODF deployment and I am not sure we have the UI deployment done for it.